README.md

```diff
@@ -58,29 +58,29 @@ While other versions often work, the above are what we officially support. YMMV!
 
 We are active and enthusiastic participants of the following community support channels:
 
-- [TezosTaquito Telegram Community Channel][telegram]
+- [ECAD Labs Discord Channel][discord]
 - [Tezos StackExchange][stackexchange]
 
 ## Project Organization
 
-`taquito` is organized as a [mono repo](https://en.wikipedia.org/wiki/Monorepo), and includes several npm packages published to npmjs.org under the `@taquito` handle. Each package has its own README and can be found in the `packages/` directory
+Taquito is organized as a [monorepo](https://en.wikipedia.org/wiki/Monorepo), and is compose of several npm packages that are [published to npmjs.org](https://www.npmjs.com/package/@taquito/taquito) under the `@taquito` handle. Each package has its own README which can be found in the corresponding directory within `packages/`.
 
 | High-Level Packages                                            | Responsibility                                               |
 | -------------------------------------------------------------- | ------------------------------------------------------------ |
 | [@taquito/taquito](packages/taquito)                           | [Facade](https://en.wikipedia.org/wiki/Facade_pattern) to lower-level, package-specific functionality  |
 
 | Low-Level Packages                                               | Responsibility                                                |
 | ---------------------------------------------------------------- | ------------------------------------------------------------- |
-| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" of Tezos operations                           |
-| [@taquito/michelson-encoder](packages/taquito-michelson-encoder) | Creates a JS abstraction for Smart Contracts                  |
-| [@taquito/michel-codec](packages/taquito-michel-codec)           | Converts Michelson between forms and expands Macros           |
+| [@taquito/local-forging](packages/taquito-local-forging)         | Local "forging" (preparation) of Tezos operations             |
```

Comment:
preparation -> serialization of Tezos operations into bytes?

Reply:
Yes, better and more clear.